testcases/kernel/syscalls/fallocate/fallocate01.c

```diff
@@ -110,8 +121,10 @@ void get_blocksize(int);
 void populate_files(int fd);
 void runtest(int, int, loff_t);
 
+char *tempdir = "tempdir";
 char *TCID = "fallocate01";
-char fname_mode1[255], fname_mode2[255];	/* Files used for testing */
+char *fname_mode1 = "tempdir/tfile_mode1";	/* Files used for testing */
```

Comment:
Commend with // is bette then /* */ for one line. 
Also you can add same comment for the second file as well